db/migrate/20220108175912_addcountry_to_games.rb

```diff
@@ -0,0 +1,4 @@
+class AddcountryToGames < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Looks like we need to add the country still. An easy fix for sure.

Reply:
The only one I didn't double check in Postico is where I messed up 🥲